app/controllers/admin_controller.rb

```diff
@@ -12,20 +12,21 @@ def recently_invalidated
   end
 
   def user_feedback
-    @feedbacks = nil
-    begin
-      @user = User.find_by_email params[:user_name]
-      @feedbacks = Feedback.unscoped.joins('inner join posts on feedbacks.post_id = posts.id').where(:user_id => @user.id).select('posts.title, feedbacks.*').order('feedbacks.id DESC').paginate(:page => params[:page], :per_page => 100)
-      @feedback_count = Feedback.unscoped.where(:user_id => @user.id).count
-      @invalid_count = Feedback.unscoped.where(:user_id => @user.id, :is_invalidated => true).count
-    rescue
+    @user = User.all.where(:id => params[:user_id])
```

Comment:
Tests failing because of this; you want `@user = User.find(params[:user_id])`. That returns a single `User`, the current way you have it returns a `[User]` (array of one).